### PR TITLE
 [textfield] If edittext gravity is center_horizontal, label when is collapsed ha…

### PR DESCRIPTION
**Pull request by marekkondracki**
If EditText gravity is center_horizontal, label when is collapsed have incorrect position. (center_vertical instead start). It resolves problem below:

[https://stackoverflow.com/questions/53413119/outlinedbox-textinputlayout-and-gravity-center/53560630](https://stackoverflow.com/questions/53413119/outlinedbox-textinputlayout-and-gravity-center/53560630)

### CODE REVIEW

**Comment by marekkondracki** [comment]
I signed it!

File: lib/java/com/google/android/material/textfield/TextInputLayout.java [hunk 1]
```diff
@@ -1051,7 +1051,7 @@ private void setEditText(EditText editText) {
 
     final int editTextGravity = this.editText.getGravity();
     collapsingTextHelper.setCollapsedTextGravity(
-        Gravity.TOP | (editTextGravity & ~Gravity.VERTICAL_GRAVITY_MASK));
+        Gravity.TOP | (Gravity.START & ~Gravity.VERTICAL_GRAVITY_MASK));
```
Comment (afohrman) [inline lib/java/com/google/android/material/textfield/TextInputLayout.java]: We'd have to check with our designers to see what our intended behavior is here. If they want to support other gravities for the EditText, we wouldn't want to hardcode the label's gravity.

I'll ask for design input. You can track this issue here: https://issuetracker.google.com/121348355.

**Comment by marekkondracki** [comment]
@afohrman, I have made proper gravity handling for Outlined Text Field.

File: lib/java/com/google/android/material/textfield/TextInputLayout.java (hunk 1 above)
Comment (marekkondracki) [reply lib/java/com/google/android/material/textfield/TextInputLayout.java]: another commit resolved this problem.

**Comment by dri94** [comment]
any updates on this? Also  need this fix

**Comment by ldjcmu** [comment]
We've recently added some suffix/prefix behavior to EditText that changes how we need this to work a bit. I'll update as soon as I have more information.

**Comment by leticiarossi** [comment]
Currently into looking into merging this!

**Pull request closed without merging**